docs/user-guide/cli-db2plugin.md

```diff
@@ -160,76 +162,74 @@ If the utility `db2connectactivate` has not been executed against the Db2 databa
 
 ## Creating a user profile
 
-You create a Db2 profile to avoid entering your connection details each time that you issue a command. You can create multiple profiles and switch between them as needed. Use one of the following methods to create a profile:
-
-- Create plug-in profiles using a configuration file: Specify your profile and connection details in the `zowe.config.json` configuration file.
+After you install the plug-in, create a Db2 profile to avoid entering your connection details each time that you issue a command. You can create multiple profiles and switch between them as needed.
 
-- Create plug-in profiles using a command: Issue the `zowe profiles create` command to create the profile.
-
-We recommend that you create profiles using the configuration file. We do not recommend using profile commands because we are removing them in a future major release.
+Specify your plug-in profile and connection details in the `zowe.config.json` configuration file.
 
 ### Creating plug-in profiles using a configuration file
 
-When you issue various `zowe config` commands, such as `init`, `auto-init`, and `convert-profiles`, they create a `zowe.config.json` configuration file. When you install the Db2 plug-in, the commands create an entry for a `db2 profile` in your `zowe.config.json` file.
+When you issue various `zowe config` commands, such as `init`, `auto-init`, and `convert-profiles`, they create a `zowe.config.json` configuration file. When you install the Db2 plug-in and then issue a command, the command creates an entry for a `db2 profile` in your `zowe.config.json` file.
 
 Alternatively, you can create a Db2 profile manually by adding a section that contains the configuration details to your `zowe.config.json` configuration file:
 
-1. Browse to the following directory: `C:\Users\<username>\.zowe`
-
-2. Open the `zowe.config.json` configuration file using a text editor or IDE, such as Visual Studio Code or IntelliJ. 
+#### Creating a Db2 profile with a command
 
-    :::note
-    
-    If the file does not exist, issue the following command to create the configuration file: `zowe config init --gc`
+1.  Install the IBM Db2 Database Plug-in for Zowe CLI.
+2.  Create a Db2 profile:
 
-    :::
+    ```
+    zowe config init
+    ```
+3.  If using a non-standard port, set the port number to your Db2 connection: **[is this step correct/applicable to Db2?]**
```

Comment:
@adam-wolfe - please respond

Reply:
I don't think we have a default Db2 port, so users would have to enter this regardless. @zFernand0

Reply:
That's correct. In Zowe V3, we added the port with a default value of 0 for the DB2 plug-in (by adding the `invludeInTemplate` option to the port without a default value)

That will force people to get prompted for a port value the first time they use a db2 plug-in command